src/gui/addnewtorrentdialog.ui

```diff
@@ -39,6 +39,9 @@
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
+          <property name="maxVisibleItems">
+           <number>10</number>
```

Comment:
No need for this. `10` is the default anyway. So revert the change on this file.